### PR TITLE
distinguish self and other card explicitly with description

### PR DESCRIPTION
**Pull request by Solcany**

### CODE REVIEW

File: assets/scss/variables.scss [hunk 1]
```diff
@@ -112,24 +113,3 @@ $warn-overlay-color: rgba(5, 5, 36, 0.3);
 $query-mobile: 450px;
 $query-mobile-landscape: 960px;
 $query-tablet: 1080px;
-
```
Comment (phivk) [inline assets/scss/variables.scss:115]: thanks for removing unused styles 👍

File: components/CardItem.vue [hunk 2]
```diff
@@ -100,13 +100,13 @@ export default {
...
-      line-height: $f-3;
+      font-size: $f-4;
+      line-height: $f-4;
     }
 
     @media (min-width: $query-mobile) and (max-width: $query-mobile-landscape) and (orientation: landscape) {
-      font-size: $f-4;
-      line-height: $f-4;
+      font-size: calc($f-5 + 0.1rem);
+      line-height: calc($f-5 + 0.1rem);
```
Comment (phivk) [inline components/CardItem.vue:109]: too bad we need such a hacky solution while maintaining our own custom type scale

File: components/CardItem.vue [hunk 3]
```diff
@@ -147,5 +147,56 @@ export default {
   &.other-card::before {
     background-image: url('@/static/image/cards/other.png');
   }
+
+  &.self-card::after,
```
Comment (phivk) [inline components/CardItem.vue:151]: all the styling for .self-card and .other-card relates to classes (and logic) in `SpectrumGameSelfOther.vue`, not in `CardItem` and would thus be better located there than here.

File: pages/ch3/index.vue [hunk 4]
```diff
@@ -7,23 +7,15 @@
           class="face-img"
           src="~/static/image/the_other.png"
         />
-        <SpectrumItemList
-          v-else="revealOtherConfirmed"
-          :spectra="otherUserProfileWithColors"
-        />
+        <SpectrumItemList v-else :spectra="otherUserProfileWithColors" />
```
Comment (phivk) [inline pages/ch3/index.vue:10]: thanks for fixing, not sure how it got in there

**Review by phivk — COMMENTED** [review]
@Solcany thanks for the additions. I have made an adjustment to distinguish:

1. your choice
2. your guess
3. the other's choice

I've left a comment about the location of .self-card and .other-card styles, but I am not requesting a change because of the project timeline (I want to merge and finalize this now).

File: components/CardItem.vue (hunk 3 above)
Comment (Solcany) [reply components/CardItem.vue:151]: That's a good point! thanks

**Pull request merged**